SeleniumPracticalExercise/PageObjects/LoginPage.cs

```diff
@@ -0,0 +1,25 @@
+﻿using OpenQA.Selenium;
+using SeleniumPracticalExercise.PageObjects.Common;
+
+namespace SeleniumPracticalExercise.PageObjects
+{
+    public class LoginPage : BasePageLocal
+    {
+        private readonly By TxtUserName = By.Name("username"); // locate username
```

Comment:
Remember when we had the discussion where we name locators as "_blahBlahLocator"? You should do that here, e.g. "_txtUserNameLocator". It makes it MUCH easier to distinguish between the locators and the method parameters if you make this change. The variable starting with a "_" is a ReSharper thing... it will flag it if you don't use it... it's an industry standard thing that we never really talked about.